refactor: convert network buttons to button element

MakTakin opened the pull request:
### Summary
There is quick refactoring [#869  ](https://github.com/OffchainLabs/arbitrum-token-bridge/issues/869)

bartlomiej94 commented on packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx:
```diff
@@ -49,7 +49,7 @@ export const NetworkSelectionContainer = ({
-                <div // TODO: replace with button
+                <button
                   key={chainId}
                   className="flex h-12 cursor-pointer flex-nowrap items-center justify-start space-x-3 px-12 text-lg font-light text-white hover:bg-[rgba(0,0,0,0.2)] lg:px-4 lg:text-base lg:font-normal lg:text-dark"
```
```suggestion
                  className="flex h-12 cursor-pointer flex-nowrap items-center justify-start space-x-3 px-12 text-lg font-light text-white hover:bg-[rgba(0,0,0,0.2)] focus-visible:ring-2 focus-visible:ring-gray-4 lg:px-4 lg:text-base lg:font-normal lg:text-dark"
```

Lets add `focus-visible:ring-2 focus-visible:ring-gray-4` to the className, so that we can see the outline when tabbing through using the keyboard.

bartlomiej94 commented on packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx:
```diff
@@ -60,7 +60,7 @@ export const NetworkSelectionContainer = ({
                   }}
-                  role="button"
+                  type="button"
                   tabIndex={i}
```
```suggestion
```

Lets also remove `tabIndex`, we don't need it anymore.

bartlomiej94 reviewed (commented):
Hey @MakTakin, thank you for your contribution! Make sure you have signed the Contributor License Agreement linked above.

I have also left some comments with tiny improvements we could still make. Thanks 😄

bartlomiej94 commented on packages/arb-token-bridge-ui/src/components/common/NetworkSelectionContainer.tsx:
```diff
@@ -49,7 +49,7 @@ export const NetworkSelectionContainer = ({
           {({ close }) => (
             <>
               {supportedNetworks?.map((chainId, i) => (
```
```suggestion
              {supportedNetworks?.map(chainId => (
```

We won't need `i` after removing `tabIndex`

MakTakin commented:
@bartlomiej94 Thanks for your comment!
I added styles to the buttons. Also I think in this case **ring-inset** looks better, so i added it.
I checked **TranferPanelMain** component. The next logical step is refactoring it. It has the same logic as **NetworkSelectionContainer**. What do you think about it?
P.s I've signed CLA, thanks!

bartlomiej94 commented:
@MakTakin Thank you for taking the time to make the changes. Based on your feedback regarding the `TransferPanelMain` I found more elements that would need refactoring for better accessibility, and created [an issue](https://github.com/OffchainLabs/arbitrum-token-bridge/issues/939) for that.

However, it seems like we need to explore it a little bit more internally to come up with the best approach to tackle this.

In the meantime, as we finish reviewing your current PR, if you're still looking for other issues I encourage you to look into our [good first issues](https://github.com/OffchainLabs/arbitrum-token-bridge/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).

Once again thank you so much for your contribution and feedback!

fionnachan commented:
Thank you for contributing @MakTakin, due to inactivity, we're closing this PR. Please feel free to check our [Contribute](https://github.com/OffchainLabs/arbitrum-token-bridge/contribute) page and work on issues that interest you in the future!
